canonical-websites/docs.ubuntu.com
# Python3 and latest run script

- Use python3 by default in local development
- Update the run script
- Build docs as part of the `./run build` phase
- Fix errors and warnings in sass file formatting
- Upgrade Travis test to use `./run` script

**NB**: As this removes the `rebuild-docs` script, the Jenkins build job will need updating to run `./run build` instead.

QA
--

``` bash
./run clean
./run  # Wait a very long time - like 10 minutes
./run test
./run serve
./run watch
```

Check everything works.

Check the Travis test passed.
